### PR TITLE
Update eagle to 8.5.1

### PR DESCRIPTION
**Pull request by FlechaMaker**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] [If the `sha256` changed but the `version` didn’t](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256),
      provide public confirmation by the developer: {{link}}

### CODE REVIEW

File: Casks/eagle.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 cask 'eagle' do
-  version '8.5.0'
-  sha256 '7058053c05fdf542ed53b15db151e362fa67147e3dc90231283be7b2efbac8e7'
+  version '8.5.1'
+  sha256 '04e9f900d75e9c13cf4b6d4a6f34dfec8d6f71fb6bb5e845c329a35a79af4870'
 
-  url "https://trial2.autodesk.com/NET17SWDLD/2017/EGLPRM/ESD/Autodesk_EAGLE_#{version}_English_Mac_64bit.pkg"
+  # eagle-updates.circuits.io was verified as official when first introduced to the cask
+  url "https://eagle-updates.circuits.io/downloads/#{version.split('.').join('_')}/Autodesk_EAGLE_#{version}_English_Mac_64bit.pkg"
```
Comment (commitay) [inline Casks/eagle.rb]: Use `#{version.dots_to_underscores}`

This isn't the current download link on the homepage, we will have to wait until it is updated.

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/eagle.rb [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+  appcast 'http://eagle.autodesk.com/eagle/release-notes',
```
Comment (commitay) [inline Casks/eagle.rb]: Please remove the `appcast` and `checkpoint`.

File: Casks/eagle.rb (hunk 1 above)
Comment (FlechaMaker) [reply Casks/eagle.rb]: Although the download link on autodesk.com haven't updated, update dialog of the EAGLE.app shows link to latest version on eagle-updates.circuits.io. Also, the link "autodesk.com/eagle" on the dialog is linked to https://eagle-updates.circuits.io/downloads/latest.html now.

screen shot 2018-01-03 at 21 43 00

**Comment by commitay** [comment]
Closing in favour of https://github.com/caskroom/homebrew-cask/pull/42688

**Pull request closed without merging**